### DISCUSSION TITLE
WearOS as collector still possible?

### DISCUSSION THREAD

**Discussion by izywizy** [NightscoutFoundation/xDrip#2457] [Q&A]
I used to use a TicWatch as a collector a few years ago until it packed up and went back to the phone as collector.

I'd really like to use a watch again so my question is it still possible to use WearOS watch as a collector ?

I know the procedure for installing apps has changed and can still be done with ADB bridge and I'm ok with that.

If it is still possible what is the current procedure ?

Would love to hear your recent experiences of this.

Regards

**Comment by izywizy** [comment]
Anyone ?  A simple yes or no would be sufficient.

**Reply by psonnera** [reply]
Actially it really depends on the sensor you use. Which is it?

**Comment by Navid200** [comment]
@psonnera

**Reply by izywizy** [reply]
Ah. thanks for response.

Euro L2 direct without any bridge like  MiaoMiao etc.

**Reply by psonnera** [reply]
No. Wear extension doesn't support L2 EU direct connection.

**Reply by izywizy** [reply]
@psonnera thanks so much, saved me splashing out on a new Watch to experiment with.

@Navid200 thanks for your help too.
